app/schemas/user.js

```diff
@@ -68,3 +68,14 @@ exports.signInSchema = {
     }
   }
 };
+
+exports.adminSchema = {
+  ...this.userSchema,
```

Comment:
do you need `this`?

Reply:
Yes! because in the userSchema i validate if user exists, but in the admin endpoint, an existing user must be updated so i don't need to validate it.

Reply:
I think you can call `userSchema` doing exports.userSchema
we use to do it that way

Reply:
You mean in using it without the `this`? The linter is not letting me do it that way

Reply:
instead of this.userSchema put exports.userSchema